src/MainUpdate.elm

```diff
@@ -497,9 +497,22 @@ update msg model =
                             |> List.map (\d -> { date = d, sessions = [] })
                             |> List.append model.datesWithSessions
                             |> List.filter (\d -> not (List.member d.date deletedDates))
+
+                    orderedDatesWithSessions =
+                        List.sortWith
+                            (\a b ->
+                                if
+                                    (a.date.year >= b.date.year && a.date.month > b.date.month)
+                                        || (a.date.month == b.date.month && a.date.day > b.date.day)
```

Comment:
I think this has an edge case bug. Wont it evaluate 02/01/2016 as greater than 01/01/2017